docs/authentication.rst

```diff
@@ -21,7 +25,41 @@ Use the OAuth token with this header:
 * After submitting the credentials on the login page, the client will
   be redirected to ``http://app-endpoint/#{token}`` the web-app.
 
-**Reading list scope**
+
+Using another flow
+::::::::::::::::::
+
+In case you need to have some configurations in order to trade your
+Firefox Account BrowserID assertion with a Bearer Token, you can use
+the ``GET /v1/fxa-oauth/params`` endpoint.
```

Comment:
In the documentation, we have no "you". Please rephrase using passive form : 

Using ``GET /v1/fxa-oauth/params``, the configuration values can be obtained. For example, in order to trade a Firefox Account BrowserID assertion for a Bearer Token.